uvloop/handles/stream.pyx

```diff
@@ -446,7 +446,6 @@ cdef class UVStream(UVBaseTransport):
             return
 
         self._maybe_pause_protocol()
-        self._loop._queue_write(self)
```

Comment:
This is actually wrong - in `_write()`, we have two early exits that don't start the `UVCheck` handler by calling `loop._queue_write()`. By moving this call out, we always start the `UVCheck` even if there is nothing to send.

Also, because of the fast-path in `_write()`, this doesn't really push all the buffers before trying to send in order to leverage the scatter sending. I'm pushing a new implementation to fix that.